css/mistica-common.css

```diff
@@ -297,6 +317,56 @@
     color: var(--vcolor-tagTextError);
 }
 
+/* Breadcrumbs */
+.mistica-breadcrumb ol {
+    display: flex;
+    flex: 1;
+    align-items: flex-end;
+    padding: 0;
+    margin: 0;
+    list-style: none;
+}
+
+.mistica-breadcrumb a {
+    text-decoration: none;
+    font-size: var(--mistica-font-size-1);
+    line-height: var(--mistica-line-height-1);
+    font-weight: var(--mistica-font-weight-1);
+    color: var(--vcolor-textPrimary);
+    position: relative;
+    padding-right: 12px;
+}
+
+.mistica-breadcrumb a::after {
```

Comment:
I think you could simplify the styles if you use `li:after` instead of `a:after` for the `/` char. You won't need `position: absolute`, reset text decoration underline, etc